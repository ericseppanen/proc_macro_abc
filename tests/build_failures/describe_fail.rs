use abc_macros::DescribeStruct;

#[derive(DescribeStruct)]
struct OhNo;

fn main() {}
