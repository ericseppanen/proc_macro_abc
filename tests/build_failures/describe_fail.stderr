error: That name is not allowed
 --> $DIR/describe_fail.rs:4:8
  |
4 | struct OhNo;
  |        ^^^^
